8.0.0 crashes with 'Cannot find module boxen.js'

I have been using this plugin happily for ages. Noticed there was 8.0.0 available, so upgraded and now the plugin throws this error when building:

```
./dev/index.ts → ./dev/dist/index.js...
(node:16660) ExperimentalWarning: The fs.promises API is experimental
[!] (plugin filesize) Error: Cannot find module '/D:/wamp/www/phaser-genesis/node_modules/rollup-plugin-filesize/dist/reporters/boxen.js'
Error: Cannot find module '/D:/wamp/www/phaser-genesis/node_modules/rollup-plugin-filesize/dist/reporters/boxen.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at D:\wamp\www\phaser-genesis\node_modules\rollup-plugin-filesize\src\index.js:125:12
    at new Promise (<anonymous>)
    at map (D:\wamp\www\phaser-genesis\node_modules\rollup-plugin-filesize\src\index.js:125:12)
    at Array.map (<anonymous>)
    at getLoggingData (D:\wamp\www\phaser-genesis\node_modules\rollup-plugin-filesize\src\index.js:121:15)
    at Promise.all.Object.keys.map.filter.map.currentBundle (D:\wamp\www\phaser-genesis\node_modules\rollup-plugin-filesize\src\index.js:158:14)
```

This is with node v10.16.0 and Rollup 2.7.6 on Windows 10.

Downgrading to filesize 7.0.0 works fine.

--- a/package.json
+++ b/package.json
@@ -12,9 +12,9 @@
   "scripts": {
     "lint": "eslint .",
     "test": "nyc ava test/index.test.js",
-    "pretest": "rollup -c",
-    "prepublishOnly": "npm run test",
-    "prebuild": "rimraf dist/*"
+    "remove-dist": "rimraf dist/*",
+    "pretest": "npm run remove-dist && rollup -c",
+    "prepare": "npm run test"
   },
   "engines": {
     "node": ">=8.0.0"

--- a/src/index.js
+++ b/src/index.js
@@ -12,6 +12,8 @@ const readFile = promisify(origReadFile);
 
 const thisDirectory = dirname(new URL(import.meta.url).pathname);
 
+const isWindows = process.platform === "win32";
+
 export default function filesize(options = {}, env) {
 	let {
 		render,
@@ -123,11 +125,26 @@ export default function filesize(options = {}, env) {
 						let p;
 						if (reporter === "boxen") {
 							p = import(
-								dirname(new URL(import.meta.url).pathname) +
-									"/reporters/boxen.js"
+								// Node should be ok with this, but transpiling
+								//  to `require` doesn't work, so detect Windows
+								//  to remove slash instead
+								// "file://" +
+								dirname(new URL(import.meta.url).pathname).slice(
+									// istanbul ignore next
+									isWindows ? 1 : 0
+								) + "/reporters/boxen.js"
 							);
 						} else {
-							p = import(pathResolve(process.cwd(), reporter));
+							p = import(
+								// Node should be ok with this, but transpiling
+								//  to `require` doesn't work, so detect Windows
+								//  to remove slash instead
+								// "file://" +
+								pathResolve(process.cwd(), reporter).slice(
+									// istanbul ignore next
+									isWindows ? 1 : 0
+								)
+							);
 						}
 						reporter = (await p).default;
 					}

